cmd/sonobuoy/app/args.go

```diff
@@ -128,6 +130,10 @@ func AddE2EConfigFlags(flags *pflag.FlagSet) *pflag.FlagSet {
 		e2eParallelFlag, defaultMode.E2EConfig.Parallel,
 		"Specify the E2E_PARALLEL flag to the conformance tests. Overrides --mode.",
 	)
+	e2eFlags.String(
+		e2eCustomRegistryFlag, "",
+		"Specify a yaml file acting as KUBE_TEST_REPO_LIST, overriding registries for test images.",
```

Comment:
TODO: Add something in the readme about this functionality and what the file looks like.